packages/builder/src/pages/builder/portal/apps/[appId]/index.svelte

```diff
@@ -78,7 +98,17 @@
       </Body>
     </div>
   {:else}
-    <iframe src={iframeUrl} title={app.name} />
+    <div class:hide={!loading} class="loading">
```

Comment:
Opting to visually "hide" the loader and the actual iframe so that stuff actually gets painted, hopefully reducing flashes.